pkg/print/stats/stats.go

```diff
@@ -32,19 +32,22 @@ func (s *Stats) FailedReconciliationSum() int {
 func (s *Stats) Handle(e event.Event) {
 	switch e.Type {
 	case event.ApplyType:
-		if e.ApplyEvent.Error != nil {
+		if e.ApplyEvent.Error != nil &&
+			e.ApplyEvent.Operation != event.Unchanged {
```

Comment:
This feels hacky, but the alternative is to add an ApplyFailed operation, which we can do in a future PR.

Reply:
Yeah, the assumption up until now has been that a non-nil error means the apply failed. When we are using the field for other uses, we should add an `ApplyFailed` operation.

Reply:
Added ApplyFailed in https://github.com/kubernetes-sigs/cli-utils/pull/570